de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/migrator/AMigrator.java

```diff
@@ -204,6 +222,34 @@ protected void removeFromTarget(ReferenceChange diff, boolean rightToLeft) {
 	protected void registerOldToNewIds() {
 	}
 	
+	@Override
+	public Set<String> getNewDependencies(Concept concept, IMigrator previousMigrator) {
+		
+		String conceptId = concept.getFullQualifiedName() + "/";
+		Concept conceptPrevious = loadConceptXmi(conceptId + previousMigrator.getResource());
+		Concept conceptNext = loadConceptXmi(conceptId + getResource());
+		
+		return getNewDependencies(conceptPrevious, concept, conceptNext);
+	}
+	
+	/**
+	 * Return new dependencies of new concept versions
+	 * @param conceptPrevious the previous concept version. A comparison between conceptPrevious
+	 * and the conceptCurrent should not yield any differences.
+	 * @param conceptCurrent the current concept as it is in the repository
+	 * @param conceptNext the next concept version
+	 * @return A set of new concept names
+	 */
+	public Set<String> getNewDependencies(Concept conceptPrevious, Concept conceptCurrent, Concept conceptNext) {

```

Comment:
Looks like we don't need `conceptPrevious`

Reply:
Yes, I just wanted to keep the interface analog to the migration... But I guess that only overcomplicates thinks